testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -4311,13 +4319,51 @@ fn partial_tenure_fork() {
     let mut last_sortition_winner: Option<u64> = None;
     let mut miner_2_won_2_in_a_row = false;
 
+    let commits_1 = signer_test.running_nodes.commits_submitted.clone();
+    let rl1_skip_commit_op = signer_test
+        .running_nodes
+        .nakamoto_test_skip_commit_op
+        .clone();
+
     let sortdb = SortitionDB::open(
         &conf.get_burn_db_file_path(),
         false,
         conf.get_burnchain().pox_constants,
     )
     .unwrap();
 
+    // Pause block commits
+    rl1_skip_commit_op.set(true);
+    rl2_skip_commit_op.set(true);
+
+    let mined_before_1 = blocks_mined1.load(Ordering::SeqCst);
+    let mined_before_2 = blocks_mined2.load(Ordering::SeqCst);
+    let commits_before_1 = commits_1.load(Ordering::SeqCst);
+    let commits_before_2 = commits_2.load(Ordering::SeqCst);
+
+    // Ensure that both block commits have been sent before continuing
```

Comment:
```suggestion
    // Mine one block before continuing
```

Reply:
I had just updated this to something similar before seeing your comment.